### PR TITLE
Added timestamps to logging

### LINKED ISSUE #38: Add timestamps to log files
The cron task (`cd /home/ubuntu/new-train-tracker && /home/ubuntu/.local/bin/pipenv run python3 -m server.history.update_history_db >> /home/ubuntu/new-train-database-updates.log`) and the mbta_api call (`new-train-tracker/server/mbta_api.py:                print(f"Writing problematic API response to {log_path}")`) both generate log files that don't have timestamps. It would be great to add timestamps to these for diagnosing issues.

### PR DESCRIPTION
**Pull request by nathan-weinberg**
Fixes #38

### CODE REVIEW

**Comment by austinjpaul** [comment]
Is there an easy way to have the times logged in local-time rather than UTC? I think that would be easier to correlate with a) MBTA events and b) Server events.

**Comment by nathan-weinberg** [comment]
@austinjpaul Done - lmk if you want any other tweaks to the formatting :+1:

File: server/mbta_api.py [hunk 1]
```diff
@@ -42,7 +44,10 @@ async def inner(some_session):
                 return json_api_doc.parse(response_json)
             except Exception as e:
                 _, log_path = tempfile.mkstemp(suffix=".log")
-                print(f"Writing problematic API response to {log_path}")
+                now = datetime.datetime.now(pytz.utc)
+                eastern = pytz.timezone('US/Eastern')
+                now_eastern = now.astimezone(eastern)
```
Comment (austinjpaul) [inline server/mbta_api.py:49]: I think this can be simplified slightly:
```
est = pytz.timezone('US/Eastern')
now = datetime.datetime.now(est)
```

**Review by austinjpaul — APPROVED** [review]
Lgtm

**Pull request merged**